devel/nsis/Portfile

```diff
@@ -23,11 +23,11 @@ distfiles           nsis-${version}-src.tar.bz2 \
                     nsis-${version}.zip
 
 checksums           nsis-3.0-src.tar.bz2 \
```

Comment:
This won't work. You submitted version for 3.01, but then check the checksums for version 3.0.

`3.0` should be replaced by `${version}` for both files.

Did you actually test the port?